02-oop-inheritance/keypoints/2_7-12/script.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Created by MaximilianoDaniel on 17/09/2015.
+ */
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+
+    //Is necessary to point to the childClass constructor
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+function augment( receivingClass, givingClass ) {
```

Comment:
Function must be stored into a variable, this will go to the global scope. WROOOONG